content/doc/book/pipeline/pipeline-as-code.adoc

```diff
@@ -80,34 +80,26 @@ dropdowns in other Jenkins jobs.
 
 == Folder Computation
 
-_Multibranch Pipeline_ projects and _Organization Folders_ extend the existing
-folder functionality by introducing 'computed' folders.  Computed folders
-automatically run a process to manage the folder contents.  This computation, in
-the case of _Multibranch Pipeline_ projects, creates child items for each
-eligible branch within the child.  For _Organization Folders_, computation
-populates child items for repositories as individual _Multibranch Pipelines_.
-
-Folder computation may happen automatically via webhook callbacks, as branches
-and repositories are created or removed.  Computation may also be triggered by
-the _Build Trigger_ defined in the configuration, which will automatically run a
-computation task after a period of inactivity (this defaults to run after one
-day).
-
+*Multibranch Pipeline* projects and *GitHub Organization Folders* extend Jenkins' folder functionality by introducing computed folders, which automatically manage their contents by dynamically creating and updating child items.
+These features require the plugin:github-branch-source[*GitHub Branch Source*] plugin to be installed, as it provides the necessary functionality for managing branches and repositories.
+ 
+* **Multibranch Pipeline Projects:** The computed folder functionality includes **Scan Multibranch Pipeline Log** and **Scan Multibranch Pipeline Now** options, which dynamically manage pipeline items. 
+These options allow Jenkins to scan repositories, generate child items for eligible branches, and automatically update the pipeline list based on changes.
+
+* **GitHub Organization Projects:** The computed folder functionality includes **Scan Organization Log** and **Scan Organization Now** options. 
+This functionality populated repositories as individual Multibranch Pipelines and provided insights into branch and repository scans.
```

Comment:
```suggestion
This functionality populates repositories as individual Multibranch Pipelines and provides insights into branch and repository scans.
```
Suggestion to make these present tense so that they match the previous sentence.